client/src/components/grants/inputs.tsx

```diff
@@ -1,43 +1,47 @@
 type InputProps = {
   label: string;
   name: string;
+  value?: string;
   changeHandler: (event: React.ChangeEvent<HTMLInputElement>) => void;
 };
 
 type RadioInputProps = {
   name: string;
-  value: boolean;
+  value: string;
+  currentValue?: string;
   changeHandler: (event: React.ChangeEvent<HTMLInputElement>) => void;
 };
 
 export function TextInput(props: InputProps) {
-  const { label, name, changeHandler } = props;
+  const { label, name, value, changeHandler } = props;
   return (
     <div style={{ display: "block" }}>
       <label htmlFor={label}>{label}</label>
       <input
+        className="appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
```

Comment:
@schultztimothy I would create a new css class that includes all those styles just to use one single name